engine/api/repositoriesmanager/dao.go

```diff
@@ -167,3 +169,29 @@ func DeleteForApplication(db gorp.SqlExecutor, app *sdk.Application) error {
 	}
 	return nil
 }
+
+//LoadLinkedApplicationNames loads applications which are linked with this repository manager name
+func LoadLinkedApplicationNames(db gorp.SqlExecutor, projectKey, rmName string) (sdk.IDNames, error) {
+	query := `SELECT application.id, application.name, application.description
+	FROM application
+		JOIN project ON project.id = application.project_id
+	WHERE project.projectkey = $1 AND application.vcs_server = $2`
+	rows, err := db.Query(query, projectKey, rmName)
```

Comment:
you should use gorp for scanning with the type `IDName`: https://github.com/ovh/cds/blob/5f39550667cd1151fdb00ab159b6652df45a82b7/sdk/common.go#L42